templates/default/onlinefs.service.erb

```diff
@@ -4,7 +4,9 @@ After = network-online.target docker.service <%= @local_dependencies %>
 Requires = docker.service <%= @local_dependencies %>
 
 [Service]
+<% if node['onlinefs']['config_dir'].nil? -%>
```

Comment:
The description of `onlinefs/config_dir` is not very clear. I suppose it contains the configuration of the external Kafka? Can you expand a bit more in metadata.rb ?